### DISCUSSION TITLE
Question: Minified React error #185 found in @inboxsdk/core@2.1.19

### DISCUSSION THREAD

**Discussion by fantaomeng** [InboxSDK/InboxSDK#1207] [General]
**Question(s)**
Minified React error #185; visit [](https://react.dev/errors/185?invariant=185)
![image](https://github.com/InboxSDK/InboxSDK/assets/88301653/cd7e4604-4f30-4b47-8e61-b764ec31d931)

**Comment by fantaomeng** [comment]
I'm using @inboxsdk/core@2.1.19, and I think this error from below code:
<img width="757" alt="image" src="https://github.com/InboxSDK/InboxSDK/assets/88301653/9b665108-0904-45c4-9780-77f2863193ed">

**Comment by wegry** [comment]
Hey @fantaomeng, a few questions:

1. Is this extension using manifest version 3 (MV3)?
1. Can you repro with the latest version of `@inboxsdk/core`?
2. Are you using React in your extension? If so, which version?
